pkg/api/v1/status/alert_configurations.go

```diff
@@ -0,0 +1,292 @@
+package status
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+type AlertConfiguration struct {
```

Comment:
All of these look like a straight copies from the lib, it might be possible (or might not be) to swap all of these definitions to something like:
```
type AlertConfiguration mongodbatlas.AlertConfiguration
```

Reply:
Doing so creates a dependency cycle

Reply:
There is a different ErrorMessage field

Reply:
Ok. We might still be able to do it for others than?
```
type Notification mongodbatlas.Notification
type Threshold mongodbatlas.Threshold
type MetricThreshold mongodbatlas.MetricThreshold
```

Reply:
The only thing that can be done this way is Notifications. And in this case, there will be no description in CRD.